overlord/fdestate/backend/reseal.go

```diff
@@ -306,11 +306,17 @@ func resealRunObjectKeys(
 	}
 
 	// list all the key files to reseal
-	keyFiles := []string{device.DataSealedKeyUnder(boot.InitramfsBootEncryptionKeyDir)}
+	keys := []secboot.KeyFileAndSlotName{
+		{
+			DevicePath: "/dev/disk/by-partlabel/ubuntu-data",
+			SlotName:   "default",
```

Comment:
nitpick, would it be better if this was a defined constant?

```go
const DefaultTokenSlotName = "default"
```

Reply:
To use the same name across disks? I do not know. The specs specifies the name for each disk. It happens to be the same. But it does not need to be. Data disk will have "default" and "default-fallback". Save disk will have "default" and "default-fallback". But that might change independently.

Reply:
could be a constant per disk (e.g. `const DefaultDataTokenSlotName`) so that if it is ever changed, we don't miss something by mistake.

Reply:
I don't have strong opinions on this, on the other hand the const name is too verbose. please feel free to ignore this suggestion.